Formula/p/polkit.rb

```diff
@@ -0,0 +1,53 @@
+class Polkit < Formula
+  desc "Toolkit for defining and handling authorizations"
+  homepage "https://github.com/polkit-org/polkit"
+  url "https://github.com/polkit-org/polkit/archive/refs/tags/125.tar.gz"
+  sha256 "ea5cd6e6e2afa6bad938ee770bf0c2cd9317910f37956faeba2869adcf3747d1"
+  license "LGPL-2.0-or-later"
+
+  bottle do
+    sha256 x86_64_linux: "812f69589b874c7617c40900b75cc00724f442f009a6537d2185b30b7e8fa141"
+  end
+
+  depends_on "gettext" => :build
+  depends_on "gobject-introspection" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "duktape"
+  depends_on "expat"
+  depends_on "glib"
+  depends_on :linux # macOS requires some patches to build due to installing into /usr/lib
```

Comment:
Would be nice to get this working on macOS. MacPorts packages this (though I think an older version).